### PR TITLE
Append note about content and format of /etc/kubernetes/cloud-config.

### PR DESCRIPTION
**Pull request by kad**
Related to discussion in kubernetes/kubernetes#39029

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2040)

### CODE REVIEW

File: docs/admin/kubeadm.md [hunk 1]
```diff
@@ -84,6 +84,9 @@ Valid values are the ones supported by `controller-manager`, namely `"aws"`,
 the cloud provider, you should create a `/etc/kubernetes/cloud-config`
 file manually, before running `kubeadm init`. `kubeadm` automatically
 picks those settings up and ensures other nodes are configured correctly.
+Format and content of the `/etc/kubernetes/cloud-config` file depends on
```
Comment (devin-donnelly) [inline docs/admin/kubeadm.md]: "The exact format and content of the file `/etc/kubernetes/cloud-config` depends on the type you specified for `--cloud-provider`; see the appropriate documentation for your cloud provider for details."

File: docs/getting-started-guides/kubeadm.md [hunk 2]
```diff
@@ -319,8 +319,8 @@ edit the `kubeadm` dropin for the `kubelet` service (`/etc/systemd/system/kubele
 If your cloud provider requires any extra packages installed on host, for example for volume mounting/unmounting, install those packages.
 
 Specify the `--cloud-provider` flag to kubelet and set it to the cloud of your choice. If your cloudprovider requires a configuration
-file, create the file `/etc/kubernetes/cloud-config` on every node and set the values your cloud requires. Also append
-`--cloud-config=/etc/kubernetes/cloud-config` to the kubelet arguments.
+file, create the file `/etc/kubernetes/cloud-config` on every node. Format and content of that file depends on your cloud requirements.
```
Comment (devin-donnelly) [inline docs/getting-started-guides/kubeadm.md]: "The exact format and content of that file depends on the requirements imposed by your cloud provider."

File: docs/getting-started-guides/kubeadm.md [hunk 3]
```diff
@@ -319,8 +319,8 @@ edit the `kubeadm` dropin for the `kubelet` service (`/etc/systemd/system/kubele
...
+Append also to the kubelet arguments `--cloud-config=/etc/kubernetes/cloud-config` if this configuration file used.
```
Comment (devin-donnelly) [inline docs/getting-started-guides/kubeadm.md]: "If you use the `/etc/kubernetes/cloud-config` file, you must append it to the kubelet arguments as follows:"

    --cloud-config=/etc/kubernetes/cloud-config

**Comment by kad** [comment]
Thanks for review @devin-donnelly, updated.

**Pull request merged**